bundles/alpha.model/src/alpha/model/transformation/reduction/SplitReduction.xtend

```diff
@@ -0,0 +1,246 @@
+package alpha.model.transformation.reduction
+
+import alpha.model.AbstractReduceExpression
+import alpha.model.AlphaExpression
+import alpha.model.AlphaInternalStateConstructor
+import alpha.model.DependenceExpression
+import alpha.model.RestrictExpression
+import alpha.model.transformation.Normalize
+import fr.irisa.cairn.jnimap.isl.ISLBasicSet
+import fr.irisa.cairn.jnimap.isl.ISLConstraint
+import fr.irisa.cairn.jnimap.isl.ISLDimType
+import fr.irisa.cairn.jnimap.isl.ISLMultiAff
+import fr.irisa.cairn.jnimap.isl.JNIPtrBoolean
+import org.eclipse.emf.ecore.util.EcoreUtil
+
+import static alpha.model.factory.AlphaUserFactory.createCaseExpression
+import static alpha.model.factory.AlphaUserFactory.createRestrictExpression
+import static alpha.model.util.AlphaUtil.*
+
+import static extension alpha.model.matrix.MatrixOperations.*
+import static extension alpha.model.util.AffineFunctionOperations.*
+import static extension alpha.model.util.AlphaUtil.copyAE
+import static extension alpha.model.util.AlphaUtil.getContainerEquation
+import alpha.model.StandardEquation
+
+/**
+ * This class carries out the analysis required for splitting from the max 
```

Comment:
Is there a link or doi number you can add for the max simplification paper? If someone needs to understand this better in the future, it would be convenient to have here.

Reply:
Yes, good idea. Done.